server/sonar-process-monitor/src/main/java/org/sonar/application/AppLogging.java

```diff
@@ -195,20 +205,20 @@ private void configureWithWrapperWritingToFile(LoggerContext ctx) {
     configureGobbler(ctx);
   }
 
-  private void configureRootWithLogbackWritingToFile(Props props, LoggerContext ctx) {
+  private void configureRootWithLogbackWritingToFile(LoggerContext ctx) {
     Logger rootLogger = ctx.getLogger(ROOT_LOGGER_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make the "rootLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312)